pages/githubissues.md

```diff
@@ -7,8 +7,16 @@
 * Work with admins to make adjustments if necessary, and merge your branch into the upstream
 
 ## Introduction
+
+On GitHub, each repository has a section where issues can be added, discussed, and fixed as a means of categorizing and addressing problems we find. Issues are often opened by our team to fix software bugs on the BeLL and in our code.
+
+**Note**: Issues are not just about adding to the wiki, but can also involve removing repetitive or unnecessary code or content. 
+
+Below, you'll find an example of an issue where I brought up a problem, researched an answer, and then fixed the issue.
```

Comment:
The only part I added was the note, saying that removing repetitive or unnecessary code or content could be a valid issue. I thought it belonged as a note in the introduction when describing what issues are or could be. Do you think I should put this note somewhere else?